Dockerfile

```diff
@@ -49,13 +54,14 @@ RUN mkdir ${PROJECT_DIR} \
 
 WORKDIR /ta-lib
 
-RUN pip install numpy==1.9.2 \
+RUN pip install 'numpy>=1.11.1,<2.0.0' \
```

Comment:
Please note that I added `  && pip install matplotlib \` to the Dockerfile in order to fix an error when executing _zipline/examples/buyapple.ipynb_ .